internal/cmd/onepasswordtemplatefuncs.go

```diff
@@ -110,5 +121,79 @@ func getOnepasswordArgs(baseArgs, args []string) []string {
 	if len(args) > 2 {
 		baseArgs = append(baseArgs, "--account", args[2])
 	}
+
 	return baseArgs
 }
+
+// refreshSession will return the current session token if the token within the environment is still valid.
+// Otherwise it will ask the user to sign in and get the new token.
+func (c *Config) onepasswordGetOrRefreshSession(callerArgs []string) string {
+	var account string
+	if len(callerArgs) > 2 {
+		account = callerArgs[2]
+	}
+
+	// maybe there's already a valid token cached in this run for this account
+	token, ok := c.Onepassword.sessionTokens[account]
+	if ok {
+		return token
+	}
+
+	var args []string
+	if account == "" {
+		// If no account has been given then look for any session tokens you can find in the environment
+		token = onepasswordInferSessionToken()
+		args = []string{"signin", "--raw"}
+	} else {
+		token = os.Getenv(fmt.Sprintf(" OP_SESSION_%s", account))
```

Comment:
nit: It looks like there's an unwanted extra space in the first argument to `fmt.Sprintf` here.

nit: As this is just string concatenation, we can use `"OP_SESSION_" + account` instead of `fmt.Sprintf`.

Reply:
Fixed.